### ISSUE TITLE
Breaking changes from 1.0.6 to 1.0.7/ or 1.1.0???

### ISSUE THREAD

**Issue by mobsean** [Urigo/graphql-scalars#297]
Hi,

I´m using Version 1.0.6 in an exisiting project.
Today I updated with npm update and **I get this error with Version 1.0.7:**

```
/foo/graphql/node_modules/graphql-scalars/index.cjs.js:540
        serialize: BigInt,
                   ^

ReferenceError: BigInt is not defined
    at BigIntFactory (/foo/graphql/node_modules/graphql-scalars/index.cjs.js:540:20)
    at Object.<anonymous> (/foo/graphql/node_modules/graphql-scalars/index.cjs.js:1223:24)
    at Module._compile (module.js:652:30)
    at Module._compile (/foo/graphql/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (module.js:663:10)
    at Object.newLoader [as .js] (/foo/graphql/node_modules/pirates/lib/index.js:104:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/foo/graphql/src/resolvers/scalars.resolver.js:1:1)
    at Module._compile (module.js:652:30)
    at Module._compile (/foo/graphql/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (module.js:663:10)
    at Object.newLoader [as .js] (/foo/graphql/node_modules/pirates/lib/index.js:104:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/foo/graphql/src/resolvers/index.js:3:1)
[nodemon] app crashed - waiting for file changes before starting...`

**and this error with Version 1.1.0.**

`/foo/graphql/node_modules/graphql-scalars/index.cjs.js:537
    BigInt.prototype.toJSON =
    ^

ReferenceError: BigInt is not defined
    at BigIntFactory (/foo/graphql/node_modules/graphql-scalars/index.cjs.js:537:5)
    at Object.<anonymous> (/foo/graphql/node_modules/graphql-scalars/index.cjs.js:1230:24)
    at Module._compile (module.js:652:30)
    at Module._compile (/foo/graphql/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (module.js:663:10)
    at Object.newLoader [as .js] (/foo/graphql/node_modules/pirates/lib/index.js:104:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/foo/graphql/src/resolvers/scalars.resolver.js:1:1)
    at Module._compile (module.js:652:30)
    at Module._compile (/foo/graphql/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (module.js:663:10)
    at Object.newLoader [as .js] (/foo/graphql/node_modules/pirates/lib/index.js:104:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/foo/graphql/src/resolvers/index.js:3:1)
[nodemon] app crashed - waiting for file changes before starting...
```

Can someone please help me?

**Comment by ardatan** [comment]
What NodeJS version are you using?

**Comment by mobsean** [comment]
node --version
v8.11.4

**Comment by ardatan** [comment]
Feel free to open a new issue if your problem persists

**Comment by mobsean** [comment]
> Feel free to open a new issue if your problem persists

node -v
v12.16.2

problem solved! thanks!
